### ISSUE TITLE
"Nested bracket padding" only effective on table-formatted elements

### ISSUE THREAD

**Issue by spazmodius** [j-brooke/FracturedJsonVSC#3]
I expected that **Nested Bracket Padding** would put spaces inside the brackets of any inlined element.  However, it seems to only do so when I also have **Table Array Minimum Similarity** <= 100.

I don't need table formatting (with proportional font it doesn't really work out).  But I _do_ want spaces inside my brackets.

For reproduction, try formatting [this JSON file](https://github.com/j-brooke/FracturedJsonVSC/files/6946511/New_York.json.txt).  Set **Nested Bracket Padding** to `true`, and try **Table Array Minimum Similarity** set to `100`, and `101`.

**Comment by j-brooke** [comment]
Right now there's no option to do what you're looking for - padded brackets with a simple inlined array.  But that does look like a good option to add.

Here's the current logic:
* Non-table, non-complex inlined arrays don't get spaces inside their brackets.  
* Table-formatted inlined arrays always get spaces inside their brackets.
* Complex inlined arrays (ones containing non-empty arrays/objects) get spaces inside the brackets according to the **Nested Bracket Padding** setting. 

The intent was to make it easier to spot simple vs. complex containers.

Example with `NestedBracketPadding=false` and `MaxInlineComplexity=3`:
```
{
    "simple": [1, 2, 3, 4, 5, 6],
    "complex": [1, 2, 3, 4, [5, 6]],
    "evenmore": [1, [2, [3]]]
}
```

And here with `NestedBracketPadding=true` and `MaxInlineComplexity=3`:
```
{
    "simple": [1, 2, 3, 4, 5, 6],
    "complex": [ 1, 2, 3, 4, [5, 6] ],
    "evenmore": [ 1, [ 2, [3] ] ]
}
```

Adding an option for bracket padding on simple arrays/objects is simple code-wise.  I'll have to think about how best to present it as an option that's easy to understand.

**Comment by j-brooke** [comment]
Implemented in version 2.2.1.
